lib/snowpack/type.ex

```diff
@@ -120,9 +120,7 @@ defmodule Snowpack.Type do
   end
 
   def encode(value, _) when is_binary(value) do
-    encoded = to_encoded_string(value)
-
-    {{:sql_varchar, String.length(encoded)}, [encoded]}
+    {{:sql_varchar, byte_size(value)}, [value]}
```

Comment:
I think `to_encoded_string` might be unused now. Let's delete it if it is.

Reply:
all of the other types still use it, so i'm going to leave it in for now